### DISCUSSION TITLE
[Windows] Unable to change color of SearchBar bottom line and selected flyout menu

### DISCUSSION THREAD

**Discussion by MattePozzy** [dotnet/maui#19476] [Q&A]
### Description

![image](https://github.com/dotnet/maui/assets/7420345/066b65ed-3791-4241-bb77-276afcd8ed51)
![image](https://github.com/dotnet/maui/assets/7420345/4493c1a6-88e5-4ff0-ba05-a970bb43512c)

These two colors (light blue in images) are unable to change, they get the color from the Windows colors theme.

### Steps to Reproduce

1. Create a MAUI project with shell with flyout menu and a searchbar
2. Try to change the colors.
3. Not possible.

### Link to public reproduction project repository

https://github.com/MattePozzy/winsearchcolorissue

### Version with bug

8.0.3

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Did you find any workaround?

No

**Comment by Vroomer** [comment]
Try `<Color x:Key="SystemAccentColor">#111111</Color>` in Platform > Windows  > App.xaml

**Comment by ghost** [comment]
Hi @MattePozzy. We have added the "s/needs-info" label to this issue, which indicates that we have an open question for you before we can take further action. This issue will be closed automatically in 7 days if we do not hear back from you by then - please feel free to re-open it if you come back to this issue after that time.

**Comment by MattePozzy** [comment]
> Try `<Color x:Key="SystemAccentColor">#111111</Color>` in Platform > Windows > App.xaml

Thank you for the reply, but this doesn't work.
This is the full code of App.xaml:

```
<maui:MauiWinUIApplication
    x:Class="SgatOfflineMaui.WinUI.App"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:maui="using:Microsoft.Maui">
    <!--RequestedTheme="Light">-->

    <maui:MauiWinUIApplication.Resources>
        <ResourceDictionary>
            <Color x:Key="SystemAccentColor">#9955AA</Color>
        </ResourceDictionary>
    </maui:MauiWinUIApplication.Resources>
</maui:MauiWinUIApplication>
```

**Comment by MattePozzy** [comment]
> Hi @MattePozzy. We have added the "s/needs-info" label to this issue, which indicates that we have an open question for you before we can take further action. This issue will be closed automatically in 7 days if we do not hear back from you by then - please feel free to re-open it if you come back to this issue after that time.

Yes sure, what kind of info do you need?
The issue is that there isn't a property to chage the colors of the SearchBar bottom line and the selected flyout menu, just run the application and see that these two colors cannot be change.
I would like to know is there is a workaround, a property that I missed, or if the feature will be implemented.

Thank you.

**Comment by PureWeen** [comment]
@MattePozzy does that property work if you set it in a standalone WinAppSDK app?

**Comment by ghost** [comment]
Hi @MattePozzy. We have added the "s/needs-info" label to this issue, which indicates that we have an open question for you before we can take further action. This issue will be closed automatically in 7 days if we do not hear back from you by then - please feel free to re-open it if you come back to this issue after that time.

**Comment by MattePozzy** [comment]
> @MattePozzy does that property work if you set it in a standalone WinAppSDK app?

Hi, thank you for the reply but I don't know it, I don't develop that kind of app, also I wouldn't even know how to do it :(
If you can tell me how to do a test with standalone WinAppSDK app I can try and reply you back. 

Thank you.
